core/config/locales/en.yml

```diff
@@ -720,7 +726,7 @@ en:
     average_order_value: Average Order Value
     avs_response: AVS Response
     back: Back
-    back_end: Backend
+    back_end: Back End
```

Comment:
I think I saw that translation used on a page so I tossed in the change but after [investigating](https://dzone.com/articles/blogging-programmers-style) it seems like the old choice of `Front End` and `Backend` isn't as strange as I thought.